app/models/subheading.rb

```diff
@@ -1,4 +1,6 @@
 class Subheading < GoodsNomenclature
+  SHORT_CODE_LENGTH = 13
```

Comment:
Is this still considered "Short" code?

Also:
It'd be great to add a comment on this like, to clarify the number 13, something like:
```
# Commodity code: 10 chars  + separator: 1 char +  Product Line Suffix: 2 chars 
```

Reply:
Have renamed it to FULL_CODE and add comment